OpenGL/Math/Matrix4.cs

```diff
@@ -108,6 +108,82 @@ public static Matrix4 Identity
             }
         }
 
+        public float this[int column, int row]
+        {
+            get
+            {
+                if (row == 0)
+                {
+                    if (column == 0) return row1.X;
```

Comment:
Once you have determined the row, why not just use `row1[column]`?  That would eliminate all of the `column` checks in this code.